src/views/Profile.vue

```diff
@@ -1,17 +1,58 @@
 <template>
   <div id="profile_page" :class="this.$store.state.css">
-    <h1>Profil utilisateur</h1>
-    <h3>Cette page est encore en construction</h3>
-    <p>Si vous avez été redirigé automatiquement ici, c'est que votre abonnement n'est pas à jour</p>
-    <p><a class="teamSelector" v-on:click="goToBillingPortal">{{$t('BILLING_MANAGEMENT')}}</a></p>
+    <div class="card-container">
+      <div class="upper-container"></div>
+      <div class="lower-container">
+        <div class="container-cadre">
+          <div class="image-container">
+           <img id="logoface" alt="Vue logoface" src="../assets/logo_round.png" />
+        </div>
+            <h3 class="ProfilName">Nom Prenom</h3>
+        <div>
+          <header>
+          <li>
+            <ul>
+              <a class="btnprofil fas fa-credit-card" v-on:click="goToBillingPortal">{{
+                $t("SUBCRIPTION")
+              }}</a>
+            </ul>
+            <ul>
+              <a class="btnprofil fas fa-book" v-on:click="goToBillingPortal">{{
```

Comment:
you should remove the onclick function since the real one isn't already implemented. A nice way to do it would be to disable the button with a tooltip stating that it is a WIP

Reply:
(same for all button except subscription and logout)